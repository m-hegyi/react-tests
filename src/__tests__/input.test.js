import React from "react";
import renderer from "react-test-renderer";
import Input from "../components/input";

it("Input render correctly", () => {
  const component = renderer.create(<Input />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
