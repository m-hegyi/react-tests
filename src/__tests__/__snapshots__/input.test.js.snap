// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input render correctly 1`] = `
<div>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</div>
`;
